packages/flutter_tools/templates/create/.travis.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+os:
+  - osx
+osx_image: xcode7.2
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git /tmp/flutter
```

Comment:
And, this script will be running in travis in a directory called something like <github-org-name>/<project-name>. We could clone the flutter repo into `../flutter; that would make the pubspec.yaml relative path work, as long as the user followed the convention of putting their projects in as sibling dirs of the flutter checkout.